cli/src/commands/workspace.rs

```diff
@@ -279,6 +282,76 @@ fn cmd_workspace_root(
     Ok(())
 }
 
+fn create_and_check_out_recovery_commit(
+    command: &CommandHelper,
+    ui: &mut Ui,
+    workspace: Workspace,
+) -> Result<Arc<ReadonlyRepo>, CommandError> {
```

Comment:
nit: `(ui, command)` instead of `(command, ui)`?

(The `workspace` parameter can be removed by switching to `workspace_helper_no_snapshot()`.)

Reply:
Done.